tests_integration/scripts/integration_account_setup.sql

```diff
@@ -0,0 +1,34 @@
+SET INT_TEST_USER = 'SNOWCLI_TEST';
+
+-- BASE SETUP
+CREATE ROLE IF NOT EXISTS INTEGRATION_TESTS;
+GRANT CREATE ROLE ON ACCOUNT TO ROLE INTEGRATION_TESTS;
```

Comment:
should we also include a line for `CREATE USER IF NOT EXISTS IDENTIFIER($INT_TEST_USER);` or do we assume that the person running the tests already has a user they want to use for the tests?

Reply:
My bad, I sent only this to Tomek. Here is the missing part:

```
create role integration_tests;
grant role integration_tests to user snowcli_test;

grant create database on account to role integration_tests;
grant create role on account to role integration_tests;
grant usage on warehouse xsmall to role integration_tests;
grant operate on warehouse xsmall to role integration_tests;

grant create application package on account to role integration_tests;
grant create application on account to role integration_tests
```

Reply:
Good point, added!